h2olog.h

```diff
@@ -23,15 +23,18 @@
 #ifndef h2olog_h
 #define h2olog_h
 
-#include <inttypes.h>
-#include <bcc/BPF.h>
+#include <cinttypes>
 #include <vector>
+#include <bcc/BPF.h>
 
 typedef struct st_h2o_tracer_t {
+    std::FILE *out;
+    std::size_t count;
```

Comment:
I'm so sorry to be annoying. Couple of things.

1) Do these need to be C++? it somewhat defeats the purpose of this structure.
2) All fields must be documented. I'm feeling guilty for not documenting the struct itself 😂

Reply:
1. No need to be C++, but it will only included by `.cc`.
2. Indeed. Will do!